wps-cloudformation-template.yaml

```diff
@@ -585,6 +585,81 @@ Resources:
                 Effect: "Allow"
                 Action: ["ses:SendRawEmail"]
                 Resource: ["*"]
+  LambdaClearBucketFunction:
```

Comment:
How about _EmptyBucketLambda_ or _EmptyBucketLambdaFunction_ ? We have been keeping the resource type at the end of the name. It is more readable that way.